<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>692</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>612</width>
    <height>692</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>612</width>
    <height>692</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GoBang</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: url(:/images/chessboard.jpg);
background-color: rgb(246, 198, 10);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pshBtnStart">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>26</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pshBtnRetract">
    <property name="geometry">
     <rect>
      <x>486</x>
      <y>634</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>retract</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>628</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rdoBtnWhite">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>white</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdoBtnBlack">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>9</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>black</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="cbbPalyType">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>26</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
